test/unit/math/mix/fun/Phi_test.cpp

```diff
@@ -2,10 +2,12 @@
 
 TEST(mathMixMatFun, Phi) {
   auto f = [](const auto& x1) { return stan::math::Phi(x1); };
-  stan::test::expect_common_unary_vectorized(f);
-  stan::test::expect_unary_vectorized(f, -27.5, 27.5);
+  stan::test::expect_common_unary_vectorized<stan::test::PromoteToComplex::No>(
+      f);
+  stan::test::expect_unary_vectorized<stan::test::PromoteToComplex::No>(
+      f, -27.5, 27.5);
   for (double x = -37.5; x <= 10; x += 0.5)
-    stan::test::expect_unary_vectorized(x);
+    stan::test::expect_unary_vectorized<stan::test::PromoteToComplex::No>(x);
```

Comment:
Is there a way to take that `PromoteToComplex::No` as a default or do we not yet have default template parameters for functions yet?

Reply:
Yeah I can switch these so that the default is no. When I started writing this my assumption was that it would be "Yes" by default

Reply:
Got it.  No need to switch.